packages/web/src/app/doc-pages/brand/illustrations-doc/illustrations-exhibit/illustrations-exhibit-details/imageDownloadUtils.ts

```diff
@@ -1,8 +1,16 @@
+import styleTokens from '@elvia/elvis-colors/dist/elvisIllustrationVariables.css';
+
 import { IllustrationColor } from '../../illustrations-data';
 
+interface Opts {
+  isPng?: boolean;
+  isDarkTheme?: boolean;
```

Comment:
Nit: It would be nice if these were not booleans. It would make it easier in case we need to support more file formats or themes in the future.